Pear/View/OnboardingSelectDropdownView.swift

```diff
@@ -109,15 +109,14 @@ class OnboardingSelectDropdownView: UIView {
 
     /// Set text of field if value already exists
     func setSelectValue(value: String) {
-        dropdownButton.setTitle("\(textTemplate) \(value)", for: .normal)
+        if value == "Grad Student" {
```

Comment:
since "Grad Student" appears in more than 1 place, might be better to make it a variable?